src/Core/TileMesh.js

```diff
@@ -41,6 +41,18 @@ class TileMesh extends THREE.Mesh {
 
         this.layerUpdateState = {};
         this.isTileMesh = true;
+
+        // Necessary event listener, to remove any child (beside OBB) attached
+        // to this tile
+        this._removedHandler = this.addEventListener('removed', () => {
+            this.removeEventListener(this._removedHandler);
+            // > 1 permits to filter tiles with only OBB
+            if (this.children.length > 1) {
+                const toRemove = [];
+                this.children.forEach((c) => { if (c.isLabel) { toRemove.push(c); } });
```

Comment:
No, it breaks the `forEach` unfortunately.

Reply:
I tested without problem, what's breacking message ?

Reply:
It is because of this line https://github.com/mrdoob/three.js/blob/master/src/core/Object3D.js#L371

By calling `splice` in a `forEach`, the array is completely rearranged, but the `forEach` doesn't know it, so you miss a test on one element. If you try this, you can see what I mean:
```js
const b = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9];
// this should remove 3 and 4 from the array
b.forEach((e, i, a) => { if (e == 3 || e == 4) { a.splice(i, 1); } });
// but if you print b, you'll see that 4 is still present
console.log(b);
```

Reply:
ok

Reply:
As I moved it in `LabelLayer`, I was able to remove the necessary selection of labels (see here https://github.com/iTowns/itowns/pull/1303/files#diff-9229f99ccb02f6c4aa8d3d42de700a92R179-R182)